code-challenges.js

```diff
@@ -41,25 +61,52 @@ var vowelTester1 = "learn"
 var vowelTester2 = "throw"
 // Expected output: 3
 
+const firstVowel = (string) => {
 
+  // what are vowels? make an array of them
+  let vowels = ['a', 'e', 'i', 'o', 'u']
 
+  for(let i = 0; i < string.length; i++) {
+    if (vowels.includes(string[i])) {
+      // return the index as soon as you find it
+      return i;
+    }
+  }
+}
 
+console.log(firstVowel(vowelTester1));
+console.log(firstVowel(vowelTester2));
```

Comment:
Fantastic! I would be interested in seeing how we could refactor this out of a for loop and utilizing a higher-order function instead.